OracleLinux/8/scripts/container-tools.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Provisioning script for the Container Tools module
+#
+# Copyright (c) 2020 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# https://oss.oracle.com/licenses/upl.
+#
+# Description: Installs the podman, buildah and skopeo Container Tools
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+echo 'Installing Container Tools module'
+
+dnf -y module install container-tools
```

Comment:
We should probably install `container-tools:2.0` here which are newer than upstream.

Reply:
Need to double check, but I think this is what is installed by default...

Reply:
True, but I'm always reticent to rely on defaults in case they change without warning. Installing a specific AppStream means we always know what version is installed and we can adjust it in future.

Reply:
Actually I was wrong, the default is `ol8` which is more recent then  `2.0`...
e.g. `ol8`has `buildah` 1.11.6-8.0.1, vs 1.11.6-7.0.1 for `2.0`

Shall we install `container-tools:ol8` ?

Reply:
Good catch, let's do that.